doc_source/console-mod7.md

```diff
@@ -38,22 +38,34 @@ In this step, you configure SoftHSM\.
    sudo su
    ```
 
-1. Create the softhsm2 configuration file in the root user's home directory\.
+1. Determine where the system-wide softhsm2.conf location is by checking the man page\. A common location is `/etc/softhsm/softhsm2.conf` but some systems may differ\.
```

Comment:
nit: Add this link here for more SoftHSM2 details: https://github.com/opendnssec/SoftHSMv2#configure-1